pkg/report/pb/data-feeds/on-chain/registry/decode.go

```diff
@@ -6,13 +6,9 @@ import (
 	"math/big"
```

Comment:
I've said this multiple times, you're not structuring the code properly. You're not doing it, and you're not using existing hints/structure but just randomly moving code - is there a pattern I'm missing?

1. `report/pb` and `report/monitor` have nothing to do with reports so I don't understand why are they in a `/report`
2. Also, why did components like `pkg/report/pb/platform/write_accepted.proto` lost the `write_target` subsection? It should be put in a specific folder `pkg/monitoring/pb/platform/write_target/write_accepted.proto` as this is it's specific domain.

Reply:
Sorry, I'm not familiar with the reasoning for the existing structure so the pattern is not obvious to me.

> have nothing to do with reports

I wouldn't say that, `report/pb` is for monitoring reports on-chian. But I understand your point, I will move them to the `pkg/monitoring` directory. 

There is already an existing `pkg/monitor` package which has some conflicting definitions relating to balance monitoring so I decided to move `report/monitor` to `pkg/monitor/beholder`, is that acceptable?

Reply:
I'm sharing context and feedback.

Considering your deadlines, I'd suggest doing what makes sense to you to get this working, and then follow up and clean up the situation.